src/Http/Headers/src/SameSiteMode.cs

```diff
@@ -1,13 +1,20 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 namespace Microsoft.Net.Http.Headers
 {
-    // RFC Draft: https://tools.ietf.org/html/draft-ietf-httpbis-cookie-same-site-00
+    /// <summary>
+    /// Determines whether to send a cookies on "same-site" or "cross-site" requests.
```

Comment:
The context for this type is slightly different from the other one (see my comments for the other one), it's for the header parser that could be used by a client or server. However it's still only ever used with response cookies, these fields aren't included with requests.

```suggestion
    /// Indicates if the client should include a cookie on "same-site" or "cross-site" requests.
```